core/server/levels.js

```diff
@@ -169,7 +169,7 @@ deleteLevel = levelId => {
 Meteor.publish('levels', function () {
   if (!this.userId) return undefined;
 
-  return Levels.find({ }, { fields: { name: 1, hide: 1, visit: 1, createdBy: 1, template: 1 } });
+  return Levels.find({ }, { fields: { name: 1, hide: 1, visit: 1, createdBy: 1, template: 1, spawn: 1 } });
```

Comment:
I don't see where the spawn attribute is used on the client side. Should we leave it anyway? It doesn't cost much so if it stays it's not a big deal

Reply:
We call "teleportToUserLevel" wich is using spawn value

Reply:
The `teleportUserInLevel` function is always called on the server side from what I have seen, so we always have access to all attributes 🤔

Reply:
Maybe, but If I don't specify { spawn: 1 } I couldn't have the value 🤷‍♂️

Reply:
On the server side it is a direct access to the database. The publish has no impact normally, it's weird 😮

Reply:
I can confirm you that we need to specify spawn: 1, I've just retested it now. And it is also used in currentLevel publish function 🤔

Reply:
Ok I see. In fact it's because the function is called on the client side in your change in the other file. If you want to avoid publishing this attribute you can do a `Meteor.call('teleportUserInLevel', levelId)`. Both are fine so I can merge if you want 🙂

Reply:
I think it's okay, you can merge :)